packages/common/src/PriceIdentifierUtils.js

```diff
@@ -6,7 +6,11 @@ const IDENTIFIER_BLACKLIST = {
 // Price identifiers that should resolve prices to non 18 decimal precision. Any identifiers
 // not on this list are assumed to resolve to 18 decimals.
 const IDENTIFIER_NON_18_PRECISION = {
-  USDBTC: 8
+  USDBTC: 8,
+  "STABLESPREAD/USDC": 6,
+  "STABLESPREAD/BTC": 8,
+  "ELASTIC_STABLESPREAD/USDC": 6,
+  BCHNBTC: 8
```

Comment:
```suggestion
  BCHNBTC: 8,
  ETHBTC: 8
```

Reply:
Hm this is tricky, because previously `ETHBTC` was used by an EMP that used DAI as collateral. Do you know that the next EMP to use this identifier will be collateralized by renBTC?

Reply:
Yeah this is tricky. There is interest in launching an ETH/BTC EMP collateralized by renBTC

Reply:
Hm let's just add the ETHBTC binding once the EMP is deployed then

Reply:
Good with this approach

Reply:
+1 -- we should let them know that they should propose an UMIP amendment.